src/cmds/print_dev_env.rs

```diff
@@ -61,22 +61,17 @@ impl PrintDevEnv {
             .await
         {
             Ok(nix_print_dev_env_exit) => nix_print_dev_env_exit,
-            err @ Err(_) => {
-                let wrapped_err = err
-                    .wrap_err_with(|| {
-                        format!(
-                            "\
-                        Could not execute `{nix_print_dev_env}`. Is `{nix}` installed?\n\n\
-                        Get instructions for installing Nix: {nix_install_url}\n\
-                        Underlying error\
-                        ",
-                            nix_print_dev_env = "nix print-dev-env".cyan(),
-                            nix = "nix".cyan(),
-                            nix_install_url = "https://nixos.org/download.html".blue().underline(),
-                        )
-                    })
-                    .unwrap_err();
-                eprintln!("{wrapped_err:#}");
+            Err(err) => {
+                let err_msg = format!(
+                    "\
+                    Could not execute `{nix_print_dev_env}`. Is `{nix}` installed?\n\n\
+                    Get instructions for installing Nix: {nix_install_url}\
+                    ",
+                    nix_print_dev_env = "nix print-dev-env".cyan(),
+                    nix = "nix".cyan(),
+                    nix_install_url = "https://nixos.org/download.html".blue().underline(),
+                );
+                eprintln!("{err_msg}\n\nUnderlying error:\n{err}", err = err.red());
```

Comment:
I'm a bit confused since according to the docs we already would print that underlying error? https://docs.rs/eyre/0.6.8/eyre/struct.Report.html#display-representations

Reply:
Yes, the change here is that it is colored red (so as to stand out more, but I'm not married to the idea and can easily revert that commit).

Reply:
I have no strong opinions about this. 😅